answer/shinooooo/JS/21_join/join.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>join</title>
+</head>
+<body>
+  <script>
+    const ary1 = ['a', 'b'];
+    const ary2 = ['c', 'd'];
+    const ary3 = ary1.concat(ary2);
+    const ary4 = ary3.join('');
+    console.log(ary4);
```

Comment:
[nits]

```suggestion
   const ary3 = ary1.concat(ary2).join('');
   console.log(ary3)
```
みたいに書いても良いかもしれません 👀